internal/server/v1/propertyvalues/state.go

```diff
@@ -71,29 +71,37 @@ func (s *state) init(
 	s.limit = limit
 	s.totalPage = map[int]int{}
 	// Read cache data based on pagination info.
-	rowListMap := map[int]cbt.RowList{}
+	prefix := bigtable.BtPagedPropValOut
+	if !arcOut {
+		prefix = bigtable.BtPagedPropValIn
+	}
+	accs := []*bigtable.Accessor{}
 	for _, c := range cursorGroup.Cursors {
 		if c != nil {
-			rowList := buildSimpleRequestRowList(entity, property, arcOut, c.GetPage())
-			rowListMap[int(c.GetImportGroup())] = rowList
+			accs = append(accs, &bigtable.Accessor{
+				ImportGroup: int(c.GetImportGroup()),
+				Body:        [][]string{{entity}, {property}, {strconv.Itoa(int(c.GetPage()))}},
```

Comment:
nit: slight more readable if making this multiple lines:

```
[][]string{
  {
    ...
  },
  ...
}
```

Reply:
Done